app/src/main/cpp/calendar.cpp

```diff
@@ -0,0 +1,17 @@
+// Write C++ code here.
```

Comment:
Do we need these `.cpp` and `.txt` files?

Reply:
Yes, they are used by calendar types, which greatly increased efficiency of our code